### PR TITLE
Um 6399 pdf

### PR DESCRIPTION
**Pull request by JithendarKumar-eGov**

### CODE REVIEW

File: core-services/pdf-service/src/EnvironmentVariables.js [hunk 1]
```diff
@@ -23,15 +23,16 @@ const envVariables = {
   DB_HOST: process.env.DB_HOST || "localhost",
   DB_NAME: process.env.DB_NAME || "PdfGen",
   DB_PORT: process.env.DB_PORT || 5432,
-  EGOV_EXTERNAL_HOST: process.env.EGOV_EXTERNAL_HOST || "https://dev.digit.org/" ,
+  EGOV_EXTERNAL_HOST: process.env.EGOV_EXTERNAL_HOST || "https://stateb.digit.org/" ,
```
Comment (Priyanka-eGov) [inline core-services/pdf-service/src/EnvironmentVariables.js:26]: revert these changes please

File: core-services/pdf-service/src/EnvironmentVariables.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
 const envVariables = {
   MAX_NUMBER_PAGES: process.env.MAX_NUMBER_PAGES || 80,
   EGOV_LOCALISATION_HOST:
-    process.env.EGOV_LOCALISATION_HOST || "http://egov-localization:8080/",
+    process.env.EGOV_LOCALISATION_HOST || "http://localhost:8071/",
```
Comment (Priyanka-eGov) [inline core-services/pdf-service/src/EnvironmentVariables.js:4]: revert these changes

File: core-services/pdf-service/src/EnvironmentVariables.js [hunk 3]
```diff
@@ -23,15 +23,16 @@ const envVariables = {
...
   STATE_SCHEMA_INDEX_POSITION_TENANTID: process.env.STATE_SCHEMA_INDEX_POSITION_TENANTID || 1,
   SAVE_PDF_DIR: process.env.SAVE_PDF_DIR || '/mnt/pdf/',
   DEFAULT_LOCALISATION_LOCALE:
     process.env.DEFAULT_LOCALISATION_LOCALE || "en_IN",
     DEFAULT_LOCALISATION_TENANT:
     process.env.DEFAULT_LOCALISATION_TENANT || "pb",
-    DATA_CONFIG_URLS: process.env.DATA_CONFIG_URLS,
-    FORMAT_CONFIG_URLS: process.env.FORMAT_CONFIG_URLS
+    DATA_CONFIG_URLS: process.env.DATA_CONFIG_URLS || "file:///home/jithendar/SanitationProject/config/dev/configs/pdf-service/data-config/property-receipt.json",
+    FORMAT_CONFIG_URLS: process.env.FORMAT_CONFIG_URLS || "file:///home/jithendar/SanitationProject/config/dev/configs/pdf-service/format-config/property-receipt.json",
```
Comment (Priyanka-eGov) [inline core-services/pdf-service/src/EnvironmentVariables.js:35]: revert these

File: core-services/pdf-service/src/utils/commons.js [hunk 4]
```diff
@@ -76,18 +76,25 @@ export const getTransformedLocale = (label) => {
...
     };
-  
+
+    //  console.log("headers------------>"+JSON.stringify(headers));
+      // console.log("url------------>"+url);
+      // console.log("request------------>"+JSON.stringify(request));
+
     let responseBody = await axios.post(url,request,headers)
     .then(function (response) {
+      console.log("Response ------------>"+JSON.stringify(response.data));
```
Comment (Priyanka-eGov) [inline core-services/pdf-service/src/utils/commons.js:90]: remove logs

File: core-services/pdf-service/src/utils/commons.js [hunk 5]
```diff
@@ -76,18 +76,25 @@ export const getTransformedLocale = (label) => {
...
         "content-type": "application/json;charset=UTF-8",
-        accept: "application/json, text/plain, */*"
+        accept: "application/json, text/plain, */*",
+        "TENANTID":envVariables.STATE_LEVEL_TENANT_ID
       }
     };
-  
+
+    //  console.log("headers------------>"+JSON.stringify(headers));
+      // console.log("url------------>"+url);
```
Comment (Priyanka-eGov) [inline core-services/pdf-service/src/utils/commons.js:85]: remove unnecessary logs and comments

File: core-services/pdf-service/src/utils/externalAPIMapping.js [hunk 6]
```diff
@@ -33,6 +34,7 @@ export const externalAPIMapping = async function (
   unregisteredLocalisationCodes,
   header
 ) {
+  console.log("qqqqqqqqqaaaa",requestInfo);
```
Comment (Priyanka-eGov) [inline core-services/pdf-service/src/utils/externalAPIMapping.js:37]: remove this

**Review by kavi-egov — APPROVED** [review]

**Review by talele08 — APPROVED** [review]
